### PR TITLE
Artifact populate db changes

### PR DESCRIPTION
**Pull request by josenavas**
This PR modifies the populate_test_db.sql file to be compliant with the new DB schema.

Fixes the first element in #1523 list.

### CODE REVIEW

File: qiita_db/support_files/populate_test_db.sql [hunk 1]
```diff
@@ -311,85 +312,76 @@ INSERT INTO qiita.prep_1 (sample_id, barcode, LIBRARY_CONSTRUCTION_PROTOCOL, pri
...
--- Link the processed data with the preprocessed data
-INSERT INTO qiita.preprocessed_processed_data (preprocessed_data_id, processed_data_id) VALUES (1, 1);
-
--- Insert the reference files for reference 1
-INSERT INTO qiita.filepath (filepath, filepath_type_id, checksum, checksum_algorithm_id, data_directory_id) VALUES
-('GreenGenes_13_8_97_otus.fasta', 10, '852952723', 1, 6),
-('GreenGenes_13_8_97_otu_taxonomy.txt', 11, '852952723', 1, 6),
-('GreenGenes_13_8_97_otus.tree', 12, '852952723', 1, 6);
+-- Insert some artifacts
+--   1 ---> 2 ---> 4
```
Comment (antgonza) [inline qiita_db/support_files/populate_test_db.sql]: Could you add an explanation of what each artifact is?

**Comment by antgonza** [comment]
One minor comment.

**Comment by ElDeveloper** [comment]
:+1:

**Comment by josenavas** [comment]
Comments addressed

**Pull request merged**